CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## 0.1.1 - 2020-06-16
+
+### Changed
+- Defaults to install as a UI Extension
+    - Read more about that [here](https://code.visualstudio.com/api/advanced-topics/remote-extensions)
+
+### Fixed
+- [Bugs fixed](https://github.com/microsoft/vscode-azurevirtualmachines/milestone/2?closed=1)
```

Comment:
Normally I'm fine with a simple link like this, but for bug-fix releases I prefer to explicitly call out the import bugs. For example:
https://github.com/microsoft/vscode-azurefunctions/blob/master/CHANGELOG.md#0201---2019-12-05

In this case, I would call out the missing ssh directory bug